docs/BacwardCompatibility.md

```diff
@@ -0,0 +1,224 @@
+entry, so we don't have to change the go structures, but will be able to validate the content of the 
+`fybrik-taxonomy-config` configuration map. Due to possibility of user modifications of the basic taxonomy file I suggest 
+in addition to the version, add the data of the file generation and maybe its CRC. 
+
+*Note:* Should we create a taxonomy json file for the Credential Management connector?
+*Note:* Should we provide a [default metric taxonomy](https://fybrik.io/v1.1/tasks/infrastructure/#add-a-new-attribute-definition-to-the-taxonomy)
+
+## Default Policies
+Fybrik is installed with a set of different default policies. Most of the change in the policies do not prevent Fybrik 
+operation, but can chnage its default behaviour and therefore influence on its users.
+
+### Default OPA Policy Rules
+The [default OPA policy rules](https://github.com/fybrik/fybrik/blob/master/charts/fybrik/files/opa-server/policy-lib/default_policy.rego) 
+defines Fybrik behavior if no other rules are not provided by users.
+
+### Default IT Config Policies 
+[IT config policies](https://fybrik.io/v1.1/concepts/config-policies/) are the mechanism via which the organization may 
+influence the construction of the data plane, taking into account infrastructure capabilities and costs.
+Out of the box policies come with the fybrik deployment. They define the deployment of basic capabilities, 
+such as read, write, copy and delete.
+The default IT policies and the default infrastructure definitions locates at 
+[./charts/fybrik/adminconfig](https://github.com/fybrik/fybrik/tree/master/charts/fybrik/files/adminconfig)
+
+*Note:* in order to update the IT config policies and define infrastructure, Fybrik 
+[recommends](https://fybrik.io/v1.1/concepts/config-policies/#how-to-provide-custom-policies) to overwrite the files in 
+the `./charts/fybrik/adminconfig` or direct changes in the `fybrik-adminconfig` configmap. However, fybrik updates will 
+remove these changes. We have to provide a different mechanism, which will automatically merge default and user defined 
+settings.  
+
+## Fybrik logs format
+Fybrik, during its running produces log files. The files are output of Fybrik execution, so we don't have to define a 
+separate version mechanism for the files format. They will be inline with the Fybrik versions. However, changing log 
+[component types](https://github.com/fybrik/fybrik/blob/b19f114988fc318819ddb6fa42059bf7e473ae63/pkg/logging/logging.go#L38)
+or log [entry parameters](https://github.com/fybrik/fybrik/blob/b19f114988fc318819ddb6fa42059bf7e473ae63/pkg/logging/logging.go#L60) 
+can break log analytics tools.   
+### Backward compatability strategy
+- Include in the Fybrik logs its version, it will help to parse historical logs.
+- Avoid changes in the log format
+- Use a graceful period (2 minor versions) for deprecated log entries
+- For renamed log entries, use both the old and teh new definitions during the graceful period. 
```

Comment:
Agreed and good points.  Would be great if there was some way in GIT to mark such key integration points as such, so that people would know to be careful before changing.  Any ideas?